plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/ChangedLinesModel.java

```diff
@@ -0,0 +1,57 @@
+package io.jenkins.plugins.coverage.metrics.model;
+
+import java.util.Objects;
+
+public class ChangedLinesModel {
+    private int startLine;
+    private int endLine;
+    private Type type;
+
+    public ChangedLinesModel(final int start_line, final int end_line, final Type type) {
+        this.setStartLine(start_line);
+        this.setEndLine(end_line);
+        this.setType(type);
+    }
+
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
These should be added in the end of the file since the getters are more of interest than equals/hashcode